Diagnostic/diagnostic.py

```diff
@@ -84,7 +84,25 @@ def init_distro_specific_actions():
     global g_dist_config, RunGetOutput
     dist = platform.dist()
     try:
-        g_dist_config = DistroSpecific.get_distro_actions(dist[0], dist[1], hutil.log)
+        if dist[0] != '':
+            name = dist[0]
+            version = dist[1]
+        else:
+            try:
+                # platform.dist() in python 2.7.15 does not recognize SLES/OpenSUSE.
+                with open("/etc/os-release") as fp:
```

Comment:
Ideally it should [fallback to /usr/lib/os-release](https://en.opensuse.org/SDB:Find_openSUSE_version) if /etc/os-release is not available.

Reply:
this PR is to enable SuSE 15; which has /etc/os-release ...

Fican-SLES15:/var/lib/waagent # ls /usr/lib/os-release
ls: cannot access '/usr/lib/os-release': No such file or directory

Some distros use /usr/lib/os-release; but usually it is a symlink to /etc/os-release